lib/install/management/delete.go

```diff
@@ -143,6 +150,46 @@ func (d *Dispatcher) DeleteVCHInstances(vmm *vm.VirtualMachine, conf *metadata.V
 	return nil
 }
 
+func (d *Dispatcher) deleteVolumeStoreIfForced(conf *metadata.VirtualContainerHostConfigSpec) error {
```

Comment:
Also please move this method to store_files.go.

And, there is logic to delete volumes directory previously, which is used before you add volume creation related logic into vic-machine.
It would be appreciated if you can clean up those logic.